tests/test_pathsupport.py

```diff
@@ -18,14 +18,14 @@ def test_addpath_data_sources_path(allkindofcomplexity):
     path = Path(allkindofcomplexity)
     data_sources = dpf.core.DataSources()
     data_sources.add_file_path(path)
-    print(data_sources)
+    # print(data_sources)
 
 
 def test_print_data_sources_path(allkindofcomplexity):
     path = Path(allkindofcomplexity)
     data_sources = dpf.core.DataSources()
     data_sources.set_result_file_path(path)
-    print(data_sources)
+    # print(data_sources)
```

Comment:
@PProfizi why is that commented. This test tests that print data sources doesn't raise

Reply:
@cbellot000 adding it back